test/unit/logs/walk-fail.log.dr

```diff
@@ -59,4 +59,4 @@ Device cleanup
        (no modifications)
    compare config
        succeeded
-Failed States  ['s-subnet3:*']
+Failed states  ['s-subnet3:*']
```

Comment:
comment related to whole test directory, not this file explicitly...
maybe rename "logs/" test tiny bit, so it does not evoke forgotten log files added to repo form other tests?

Reply:
Makes sense, but I'll do that separately, otherwise I would pollute this already bit too long PR with unrelated stuff.